File: frontend/app/src/main/java/com/capstone/whereigo/ItemSpacingDecoration.java
package com.capstone.whereigo;

import android.graphics.Rect;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

// RecyclerView에 padding(간격) 넣으려고 쓴 클래스
public class ItemSpacingDecoration extends RecyclerView.ItemDecoration {

    private final int space;

    public ItemSpacingDecoration(int space) {
        this.space = space;
    }

    @Override
    public void getItemOffsets(@NonNull Rect outRect,
                               @NonNull View view,
                               @NonNull RecyclerView parent,
                               @NonNull RecyclerView.State state) {
        outRect.bottom = space;
    }
}

File: frontend/app/src/main/java/com/capstone/whereigo/ItemSpacingDecoration.kt
package com.capstone.whereigo

import android.graphics.Rect
import android.view.View
import androidx.recyclerview.widget.RecyclerView

// Recycle view 에 padding 넣으려고 쓴거
class ItemSpacingDecoration(private val space: Int) : RecyclerView.ItemDecoration() {
    override fun getItemOffsets(
        outRect: Rect,
        view: View,
        parent: RecyclerView,
        state: RecyclerView.State
    ) {
        outRect.bottom = space
    }
}
